dpnp/dpnp_iface_mathematical.py

```diff
@@ -2576,7 +2594,8 @@ def gradient(f, *varargs, axis=None, edge_order=1):
     Input array, expected to have numeric data type.
 out : {None, dpnp.ndarray, usm_ndarray}, optional
     Output array to populate.
-    Array must have the correct shape and the expected data type.
+    Array must have a shape that the input broadcast to and
+    the expected data type.
```

Comment:
This one was missed.

```suggestion
    Array must have the correct shape and the expected data type.
```